.github/workflows/ci.js.yml

```diff
@@ -29,7 +29,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: ['16', '17', '18']
+        node-version: ['18', '19', '20']
```

Comment:
Since 19 is unstable, I don't think we should test for it at all (same for 17, 21, etc.)